modules/info-list/info-list.php

```diff
@@ -66,6 +66,7 @@ public function get_icon( $icon = '' ) {
 	 * @param object $settings gets the settings for the module.
 	 */
 	public function render_image( $item, $settings ) {
+		$settings->icon_image_size = isset( $settings->icon_image_size ) ? $settings->icon_image_size : 75;
```

Comment:
In this code, we are receiving error due to undefined $infolist_icon_size  variable. It is not defined out of if-else statement.

So, instead of this - 

$infolist_icon_size = isset( $settings->icon_image_size ) ? $settings->icon_image_size : 75;

And replace $settings->icon_image_size on all the places with $infolist_icon_size 

Like this - 

$infolist_icon_size = isset( $settings->icon_image_size ) ? $settings->icon_image_size : 75;

if ( 'circle' == $settings->list_icon_style ) {
    if ( is_numeric( $infolist_icon_size ) ) {
        $infolist_icon_size = $infolist_icon_size  / 2;
    }
} elseif ( 'square' == $settings->list_icon_style ) {
    if ( is_numeric( $infolist_icon_size ) ) {
        $infolist_icon_size = $infolist_icon_size / 2;
    }
} elseif ( 'custom' == $settings->list_icon_style ) {
    $infolist_icon_size = $infolist_icon_size;
} else {
    $infolist_icon_size = $infolist_icon_size;
}

Reply:
Okay 
Done